### ISSUE TITLE
-p option rejected by variants

### ISSUE THREAD

**Issue by mmiladi** [jts/nanopolish#393]
Hello,
There seems to be an inconsistency for `nanopolish variants` between `--ploidy` and its short form `-p`

```
$ nanopolish variants --consensus polished.fa -r reads.fasta -b reads.bam -g genome.fa -w "tig00000001:200000-202000" --ploidy 1
##fileformat=VCFv4.2
##INFO=<ID=TotalReads,Number=1,Type=Integer,Description="The number of event-space reads used to call the variant">
##INFO=<ID=SupportFraction,Number=1,Type=Float,Description="The fraction of event-space reads that support the variant">
...
```
vs: 

```
nanopolish variants --consensus polished.fa -r reads.fasta -b reads.bam -g genome.fa -w "tig00000001:200000-202000" -p 1
variants: invalid option -- 'p'
variants: too many arguments
Usage: nanopolish variants [OPTIONS] --reads reads.fa --bam alignments.bam --genome genome.fa
..
```

```
$ nanopolish --version
nanopolish version 0.9.0

```

**Comment by jts** [comment]
Hi @mmiladi,

This was fixed already (see #323) and should work in `0.9.1`. This version isn't in conda yet but hopefully will be soon.

Jared

**Comment by mmiladi** [comment]
Hi Jared,

Thanks much for the quick update.

Milad
